okta/table_okta_trusted_origin.go

```diff
@@ -53,31 +53,55 @@ func listOktaTrustedOrigins(ctx context.Context, d *plugin.QueryData, _ *plugin.
 	logger := plugin.Logger(ctx)
 	client, err := Connect(ctx, d)
 	if err != nil {
-		logger.Error("listOktaTrustedOrigins", "connect", err)
+		logger.Error("listOktaTrustedOrigins", "connect_error", err)
 		return nil, err
 	}
 
-	input := query.Params{}
+	// Maximum limit isn't mentioned in the documentation
+	// Default maximum limit is set as 1000
+	input := query.Params{
+		Limit: 1000,
```

Comment:
What's the reasoning for picking 1000 as the default?

Reply:
In the other plugins, it was decided to set 1000 as the maximum limit when it is not mentioned in the documentation. Then we will test the query manually and reduce the number if the API doesn't support it. I have followed the same pattern in this case.

Reply:
It's good to look at other plugins, but we should probably look at Okta's other limits, not other plugins, to base a default off of. What are most defaults for other Okta APIs? 200?

Reply:
In most of the cases that I have come across so far, it's 200. I have 10000 and 500 in 2 tables.

Reply:
Let's go with 200 then, it feels like a safe bet based on other APIs - if we come across different information later on, we can update the default.

Reply:
I have updated the default limits of both `okta_network_zone` and `okta_trusted_origin` to 200.